src/test/java/reactor/kafka/receiver/KafkaReceiverTest.java

```diff
@@ -1357,16 +1359,17 @@ private void cancelSubscriptions(boolean failOnError) {
         subscribeDisposables.clear();
     }
 
-    private int committedCount(KafkaReceiver<Integer, String> receiver) {
-        return receiver.doOnConsumer(c -> {
-            int committed = 0;
-            for (int j = 0; j < partitions; j++) {
-                TopicPartition p = new TopicPartition(topic, j);
-                OffsetAndMetadata offset = c.committed(p);
-                if (offset != null && offset.offset() > 0)
-                    committed += offset.offset();
-            }
-            return committed;
+    private long committedCount(KafkaReceiver<Integer, String> receiver) {
+        return receiver.doOnConsumer(consumer -> {
+            Set<TopicPartition> topicPartitions = IntStream.range(0, partitions)
```

Comment:
is this change really necessary? Did they remove `committed(TopicPartition)`?

Reply:
`OffsetAndMetadata committed(TopicPartition partition)` is deprecated